sdk/keyvault/azure-keyvault-securitydomain/tests/test_security_domain.py

```diff
@@ -20,9 +21,13 @@ class TestSecurityDomain(KeyVaultTestCase):
     def test_security_domain_download_and_upload(
         self, client: SecurityDomainClient, upload_client: SecurityDomainClient, **kwargs
     ):
-        # Before running this test, create security domain certificates
-        # 1. Create private keys: `openssl genrsa -pubout -out <>-certificate[0-2].key 2048`
-        # 2. Create certificates: `openssl req -new -x509 -days 365 -key <>-certificate[0-2].key -out <>-certificate[0-2].cer`
+        """Test downloading a security domain and uploading it to another Managed HSM.
+
+        This test requires two Managed HSMs to be set up, one for downloading and one for uploading. The URL of the
+        first Managed HSM is specified in the environment variable AZURE_MANAGEDHSM_URL, and the second is specified
+        in SECONDARY_MANAGEDHSM_URL.
```

Comment:
From what I could see, the util code that read from these env vars was removed? Are they read again somewhere else?